src/internet_identity/tests/integration/stable_memory.rs

```diff
@@ -300,30 +300,3 @@ fn should_trap_on_old_stable_memory() -> Result<(), CallError> {
     }
     Ok(())
 }
-
-/// Tests that II will refuse to upgrade on stable memory without persistent state.
-#[test]
-fn should_trap_on_missing_persistent_state() -> Result<(), CallError> {
```

Comment:
`v9` persistent state exists by definition. Hence a back-up without one can no longer be created.